pkg/helpers/describe/describer.go

```diff
@@ -2086,6 +2095,79 @@ func describeSecurityContextConstraints(scc *securityv1.SecurityContextConstrain
 	})
 }
 
+// MachineDescriber generates information about a machine
+type MachineDescriber struct {
+	restClient *rest.RESTClient
+}
+
+func (d *MachineDescriber) Describe(namespace, name string, settings describe.DescriberSettings) (string, error) {
+	machineObj := &machinev1beta1.Machine{}
+
+	err := d.restClient.Get().
```

Comment:
Machine client is missing in the openshift/client-go. Until we have that, I am doing the request manually.

Reply:
resolved